tracker-agent/src/main/configuration/tracker-agent-server.properties

```diff
@@ -7,23 +7,24 @@
 # for in your License Agreement with Software AG.
 #
 
-C8Y.baseURL=http://localhost:8111
+# the url of the Cumulocity platform
+C8Y.baseURL=http://cumulocity:8111
 C8Y.forceInitialHost=true
 
+# used for requesting device credentials, ROLE_DEVICE_BOOTSTRAP is required
 C8Y.devicebootstrap.tenant=management
 C8Y.devicebootstrap.user=devicebootstrap
 C8Y.devicebootstrap.password=Fhdt1bb1f
 
+# required for integration tests
 C8Y.bootstrap.tenant=management
 C8Y.bootstrap.user=servicebootstrap_tracker-agent
-C8Y.bootstrap.password=
-C8Y.microservice.isolation=MULTI_TENANT
+C8Y.bootstrap.password=CNFww5Mf2Tj0L35S6rb3YjezF2F0nCkU
```

Comment:
I am confused about the comment above: required for integration tests. Isn't this also required for agent runtime?
Another thing, this CNFww5Mf2Tj0L35S6rb3YjezF2F0nCkU value will work only in your env.. Have you added as example?

Reply:
Sorry, you are right, it might be needed in runtime! It is just I run tracker-agent as a regular microservice. That's why it worked even if the credentials were wrong.

I'll change the comment.

Still the whole file is used only in the integration test. Runtime is reading it from same file in /etc/tracker-agent/.